pandas/core/tools/datetimes.py

```diff
@@ -508,6 +508,10 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
         from pandas import Series
         values = _convert_listlike(arg._values, False, format)
         result = Series(values, index=arg.index, name=arg.name)
+        if utc and isinstance(values, np.ndarray):
```

Comment:
I'm not sure how to include this logic in convert_listlike as is.

`._values` is called on `Series` before it's passed into convert_listlike which turns the `Series` into a `np.array`, so convert_listlike doesn't "know" that a `Series` was the original argument.

Not sure why this was done originally, but I could have convert_listlike handle a `Series` directly?